docs/release_process.md

```diff
@@ -40,8 +40,18 @@ Let's make sure there are no migration conflicts with the previous release:
     $ git checkout master
     $ make migrate
```

Comment:
Do we keep those lines ?
This documentation version will be in the 0.22.0, we don't need to do (and document) manual migration tests with imperfect data while we have a automated migration test with better data.